src/mcts/search.h

```diff
@@ -271,13 +271,9 @@ class SearchWorker {
                                    int collision_count) {
       return NodeToProcess(node, depth, true, collision_count);
     }
-    static NodeToProcess Extension(Node* node, uint16_t depth) {
+    static NodeToProcess Single(Node* node, uint16_t depth) {
```

Comment:
It feels that ther should be a better name than `Single` but I cannot come up with it.
`TerminalHit`, `Collision`, and `Extension` hinted what will happen with this node, while `Single` just means that's no multivisit.. Maybe `Visit`? `LeafVisit`? `LeafEval`? `Eval`? `Backprop`? `Process`? :)

Reply:
Renamed to `Visit` (as Leaf implies no children but "terminals" might have children in the tree).